components/wrapping-modal.tsx

```
@@ -0,0 +1,20 @@
+import React from 'react'
+
+type WrappingModalProps = {
+  closeModal: () => void
+}
+
+const WrappingModal: React.FC<WrappingModalProps> = ({ closeModal }) => {
```

ESLint

[error] 'closeModal' is defined but never used.

(@typescript-eslint/no-unused-vars)